### ISSUE TITLE
Update versions of dependencies

### ISSUE THREAD

**Issue by ageorgou** [SilverLabUCL/PySilverLabNWB#11]
Requiring `numpy < 1.14` causes problems for other dependencies:

```
ERROR: imagecodecs-lite 2019.12.3 has requirement numpy>=1.14.5, but you'll have numpy 1.13.3 which is incompatible.
ERROR: tifffile 2019.7.26.2 has requirement numpy>=1.14.6, but you'll have numpy 1.13.3 which is incompatible.
```

**Comment by alessandrofelder** [comment]
Changed in 28bcb116030e63bbb81495524f7e17bd08c8e565

**Comment by alessandrofelder** [comment]
possibly fixed in 5898a0f38cd3842673c452dfc569b1f19fd5ecfb ?

**Comment by ageorgou** [comment]
Done in #13.
